src/test/java/seedu/sellsavvy/logic/commands/ordercommands/OrderCommandTestUtil.java

```diff
@@ -48,6 +49,16 @@ public class OrderCommandTestUtil {
     public static final String INVALID_COUNT_NEGATIVE = " " + PREFIX_COUNT + "-2";
     public static final String INVALID_COUNT_STRING = " " + PREFIX_COUNT + "2 some random string";
 
+    public static final EditOrderCommand.EditOrderDescriptor DESC_ATLAS;
```

Comment:
Can this be renamed to `EDIT_DESC_ATLAS` and `EDIT_DESC_BOTTLE` instead for clarity?